CRM/Contribute/BAO/Contribution.php

```diff
@@ -4340,6 +4340,16 @@ public static function completeOrder($input, $ids, $contribution, $isPostPayment
     $contribution->contribution_status_id = $contributionParams['contribution_status_id'];
 
     CRM_Core_Error::debug_log_message('Contribution record updated successfully');
+    $contributionSoft = civicrm_api3('ContributionSoft', 'get', [
```

Comment:
Since the v4 api for ContributionSoft is about to be merged let's use that

https://github.com/civicrm/civicrm-core/pull/19083

Reply:
Thanks @eileenmcnaughton 
I will update the code to use api4 then :)